howso/editing.amlg

```diff
@@ -1,14 +1,112 @@
 ;Contains methods for adding, editing and removal of cases and session data.
 (null
+	;Cleans up trainee data, removing unused sessions, cases or actions missing data, etc. given the following optional parameters.  If a parameter is not specified, it will look to this entity's own label of the same name.  See label for parameter details.
+	#clean_data
```

Comment:
I think this label got removed in main, might want to merge main when possible

Reply:
Yup, doing this today.

Reply:
Removed in next commit